transport/EndpointMetrics.h

```diff
@@ -0,0 +1,30 @@
+#include <cstdint>
+
+#pragma once
+
+struct EndpointMetrics
+{
+    EndpointMetrics(uint32_t rQueueCount, uint32_t sQueueCount)
+        : rxQueue(rQueueCount)
+        , txQueue(sQueueCount)
+    { }
+
+    EndpointMetrics() : EndpointMetrics(0, 0) { }
+
+    EndpointMetrics& operator+=(const EndpointMetrics& rhs)
+    {
+        rxQueue += rhs.rxQueue;
+        txQueue += rhs.txQueue;
+        return *this;
+    }
+
+    uint32_t rxQueue;
```

Comment:
we should add sendQueue here

Reply:
I added, it's the txQueue. I can change the name if you prefer

Reply:
Ok. I get it now